qiskit/aqua/circuits/boolean_logical_circuits.py

```diff
@@ -245,11 +245,11 @@ def _construct_circuit_for_tiny_expr(self, circuit, output_idx=0):
         if self._ast == ('const', 0):
             pass
         elif self._ast == ('const', 1):
-            circuit.u3(pi, 0, pi, self._output_register[output_idx])
+            circuit.u(pi, 0, pi, self._output_register[output_idx])
```

Comment:
E.g. this is just an X gate 🙂 
```suggestion
            circuit.x(self._output_register[output_idx])
```